mesh/src/main/java/no/nordicsemi/android/mesh/transport/GenericLocationGlobalSetUnacknowledged.java

```diff
@@ -0,0 +1,97 @@
+package no.nordicsemi.android.mesh.transport;
+
+
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+import no.nordicsemi.android.mesh.ApplicationKey;
+import no.nordicsemi.android.mesh.data.GlobalAltitude;
+import no.nordicsemi.android.mesh.data.GlobalLatitude;
+import no.nordicsemi.android.mesh.data.GlobalLongitude;
+import no.nordicsemi.android.mesh.opcodes.ApplicationMessageOpCodes;
+import no.nordicsemi.android.mesh.utils.SecureUtils;
+
+/**
+ * To be used as a wrapper class for when creating the GenericLocationGlobalSetUnacknowledged message.
+ */
+public final class GenericLocationGlobalSetUnacknowledged extends ApplicationMessage {
```

Comment:
Couldn't you extend the GenericLocationGlobalSet class instead of duplicating the same code? Oh well, I guess it's the same all over the SDK.

Reply:
SDK convention should decide this I think.